### PR TITLE
update learn page

### PR DESCRIPTION
**Pull request by jmcook1186**
## Description

Minor changes overall. Swaps or removes outdated links for more recent ones, plus some tweaks to wording to clarify concepts.

### CODE REVIEW

**Comment by konopkja** [comment]
looks good to me, would add few adjustment requests:
Screenshot 2023-07-20 at 9 33 11

File: src/intl/en/page-learn.json [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+  "toc-learn-about-the-ethereum-protocol": "Learn about the Ethereum protocol",
+  "toc-learn-about-the-ethereum-community": "Learn about the Ethereum community",
   "toc-books-and-podcasts": "Books and podcasts",
   "hero-header": "Learn about Ethereum",
   "hero-subtitle": "Your educational guide to the world of Ethereum. Learn how Ethereum works and how to connect to it. This page includes technical and non-technical articles, guides, and resources.",
   "hero-button-lets-get-started": "Let's get started",
-  "what-is-crypto-1": "You may have heard about cryptocurrencies, Bitcoin, and blockchain. This page will help you learn about what they are how they relate to Ethereum.",
-  "what-is-crypto-link-1": "And what is Ethereum anyway?",
-  "what-is-crypto-2": "Bitcoin enables anyone to transfer money globally. Ethereum does too, <b>and</b> it enables anyone to create and agree on any kind of contract they can imagine. It’s both resilient and flexible: any computer program can run on Ethereum.",
+  "what-is-crypto-1": "You may have heard about cryptocurrencies, blockchains and Bitcoin. The links below will help you to learn what they are and how they relate to Ethereum.",
```
Comment (minimalsm) [inline src/intl/en/page-learn.json:13]:
```suggestion
  "what-is-crypto-1": "You may have heard about cryptocurrencies, blockchains and Bitcoin. The links below will help you learn what they are and how they relate to Ethereum.",
```

Comment (minimalsm) [reply src/intl/en/page-learn.json:13]: Don't really see the point in this change, but it's whatever 🤷‍♀️

File: src/intl/en/page-learn.json [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
   "what-is-ethereum-card-description": "If you are new, start here to learn why Ethereum matters.",
   "what-is-ethereum-card-image-alt": "Illustration of a person peering into a bazaar, meant to represent Ethereum.",
   "what-is-eth-card-title": "What is ETH?",
   "what-is-eth-description": "Ether (ETH) is the currency powering the Ethereum network and apps.",
-  "where-can-i-get-eth-card-title": "Where can I get ETH?",
-  "where-can-i-get-eth-card-description": "There are many ways to get ETH, depending on your location.",
+  "what-is-web3-card-title": "What is Web3?",
+  "what-is-web3-card-description": "Web3 is a model for the internet valuing ownership of your own assets and identity.",
   "additional-reading-more-on-ethereum-basics": "More on Ethereum basics",
+  "guides-hub-desc": "Guides: step-by-step instructions on how to use Ethereum",
```
Comment (minimalsm) [inline src/intl/en/page-learn.json:23]: cc: @konopkja 

```suggestion
  "guides-hub-desc": "Guides: step-by-step instructions on using Ethereum",
```

File: src/intl/en/page-learn.json [hunk 3]
```diff
@@ -1,37 +1,37 @@
...
+  "quiz-hub-desc": "Quiz hub: test your knowledge and learn more",
```
Comment (minimalsm) [inline src/intl/en/page-learn.json:24]: cc: @konopkja 

```suggestion
  "quiz-hub-desc": "Quiz hub: test your knowledge",
```

File: src/intl/en/page-learn.json [hunk 4]
```diff
@@ -1,37 +1,37 @@
...
   "additional-reading-what-are-smart-contracts": "What are smart contracts?",
-  "additional-reading-a-developers-intro": "An introduction to Ethereum for developers",
   "additional-reading-what-is-web3": "What is web3?",
-  "additional-reading-value": "What is value?",
-  "additional-reading-decentralize-everything": "Decentralize everything",
-  "how-do-i-use-ethereum-1": "Using Ethereum is like moving to a new place. Things will seem unfamiliar at first, and everyone moves for their own reasons. Despite this, the first thing you'll need are new keys. The software that creates and holds the keys to your new home is called a \"wallet\".",
+  "additional-reading-decentralize-everything": "Video: Explanation of what Ethereum is by Vitalik Buterin",
```
Comment (minimalsm) [inline src/intl/en/page-learn.json:27]: I'd use [this devcon talk](https://www.youtube.com/watch?v=UihMqcj-cqc) and change the copy here to 'Ethereum in 30 minutes - Vitalik Buterin'

cc: @konopkja 

```suggestion
  "additional-reading-decentralize-everything": "Video: Explanation of what Ethereum is by Vitalik Buterin",
```

File: src/intl/en/page-learn.json [hunk 5]
```diff
@@ -1,37 +1,37 @@
...
+  "what-is-crypto-1": "You may have heard about cryptocurrencies, blockchains and Bitcoin. The links below will help you to learn what they are and how they relate to Ethereum.",
+  "what-is-crypto-2": "Cryptocurrencies, such as Bitcoin, enable anyone to transfer money globally. Ethereum does too, but it can also run code that enables people to create apps and organizations. It’s both resilient and flexible: any computer program can run on Ethereum. Learn more and find out how to get started:",
   "what-is-ethereum-card-title": "What is Ethereum?",
   "what-is-ethereum-card-description": "If you are new, start here to learn why Ethereum matters.",
   "what-is-ethereum-card-image-alt": "Illustration of a person peering into a bazaar, meant to represent Ethereum.",
   "what-is-eth-card-title": "What is ETH?",
   "what-is-eth-description": "Ether (ETH) is the currency powering the Ethereum network and apps.",
-  "where-can-i-get-eth-card-title": "Where can I get ETH?",
```
Comment (minimalsm) [inline src/intl/en/page-learn.json:21]: Don't understand this change. This is one of the top questions new users have and we shouldn't bury it in the 'other links' section IMO.

File: src/intl/en/page-learn.json (hunk 4 above)
Comment (jmcook1186) [reply src/intl/en/page-learn.json:27]:
```suggestion
  "additional-reading-decentralize-everything": "Ethereum in 30 minutes by Vitalik Buterin",
```

File: src/intl/en/page-learn.json [hunk 6]
```diff
@@ -1,37 +1,37 @@
...
-  "additional-reading-a-developers-intro": "An introduction to Ethereum for developers",
   "additional-reading-what-is-web3": "What is web3?",
-  "additional-reading-value": "What is value?",
-  "additional-reading-decentralize-everything": "Decentralize everything",
-  "how-do-i-use-ethereum-1": "Using Ethereum is like moving to a new place. Things will seem unfamiliar at first, and everyone moves for their own reasons. Despite this, the first thing you'll need are new keys. The software that creates and holds the keys to your new home is called a \"wallet\".",
+  "additional-reading-decentralize-everything": "Video: Explanation of what Ethereum is by Vitalik Buterin",
+  "additional-reading-get-eth": "Learn how to get ETH",
+  "how-do-i-use-ethereum-1": "Using Ethereum can mean lots of things to lots of people. Maybe you want to sign in to an app, prove your online identity, participate in a decentralized community, transfer some ETH or swap some tokens. Maybe you want to build something that anyone can use. The first thing you'll need is an account. The easiest way to create and access an account is using software called a \"wallet\".",
```
Comment (minimalsm) [inline src/intl/en/page-learn.json:29]: > Maybe you want to sign in to an app, prove your online identity, participate in a decentralized community, transfer some ETH or swap some tokens. Maybe you want to build something that anyone can use.

Too much info. [Pick three](https://en.wikipedia.org/wiki/Rule_of_three_(writing))

> \"wallet\".",

why are we surrounding wallet in quotes here? Should we just remove?

```suggestion
  "how-do-i-use-ethereum-1": "Using Ethereum can mean lots of things to lots of people. Maybe you want to sign in to an app, prove your online identity, participate in a decentralized community, transfer some ETH or swap some tokens. Maybe you want to build something that anyone can use. The first thing you'll need is an account. The easiest way to create and access an account is using software called a wallet.,
```

File: src/intl/en/page-learn.json [hunk 7]
```diff
@@ -1,37 +1,37 @@
...
   "what-is-a-wallet-card-title": "What is a wallet?",
-  "what-is-a-wallet-card-description": "Wallets are like keyrings: they store what you need to get access to the places you value.",
+  "what-is-a-wallet-card-description": "Digital wallets are similar to physical ones; they store what you need to prove who you are and get access to the places you value.",
```
Comment (minimalsm) [inline src/intl/en/page-learn.json:31]: Some brainstorming

```suggestion
  "what-is-a-wallet-card-description": "Digital wallets are like physical wallets; they store what you need to prove who you are and get access to the places you value.",
```

Or maybe

```suggestion
  "what-is-a-wallet-card-description": "Digital wallets are like real wallets; they store what you need to prove who you are and get access to the places you value.",
```

Second sentence

```suggestion
  "what-is-a-wallet-card-description": "Digital wallets are like real wallets; they store what you need to prove your identity and get access to the places you value.",
```

Maybe too far?

```suggestion
  "what-is-a-wallet-card-description": "Digital wallets are like real wallets. They keep important stuff that shows who you are and lets you use what you care about.",
```

File: src/intl/en/page-learn.json [hunk 8]
```diff
@@ -1,37 +1,37 @@
...
   "what-is-a-wallet-card-alt": "Illustration of a robot.",
   "find-a-wallet-card-title": "Find a wallet",
-  "find-a-wallet-card-description": "We'll help you find the best wallet based on the features that matter to you.",
+  "find-a-wallet-card-description": "Browse popular wallets based on the features that matter to you.",
```
Comment (minimalsm) [inline src/intl/en/page-learn.json:34]: The wallets we show aren't 'popular'

```suggestion
  "find-a-wallet-card-description": "Browse wallets based on the features that matter to you.",
```

File: src/intl/en/page-learn.json [hunk 9]
```diff
@@ -111,29 +107,17 @@
...
-  "the-truth-machine-title": "The Truth Machine",
-  "the-truth-machine-description": "February 27, 2018 - Paul Vigna, Michael J. Casey",
-  "digital-gold-title": "Digital Gold",
-  "digital-gold-description": "May 24, 2021 - Nathaniel Popper",
-  "kings-of-crypto-title": "Kings of Crypto",
-  "kings-of-crypto-description": "December 15, 2020 - Jeff John Roberts",
   "mastering-ethereum-title": "Mastering Ethereum",
   "mastering-ethereum-description": "December 23, 2018 – Andreas M. Antonopoulos, Gavin Wood Ph.D.",
-  "podcasts-about-ethereum": "Podcasts about Ethereum and Cryptocurrencies",
+  "podcasts-about-ethereum": "Podcasts about Ethereum",
```
Comment (minimalsm) [inline src/intl/en/page-learn.json:112]: Don't think the ordering is optimal here but weakly held

File: src/intl/en/page-learn.json (hunk 5 above)
Comment (jmcook1186) [reply src/intl/en/page-learn.json:21]: The cards are supposed to link to content that answer the question in their subheading: "what is Ethereum?" 
The three new cards all contribute to the answer. The 'Get ETH' page doesn't - it tells people how to get it but doesn't help them understand what it is.

File: src/intl/en/page-learn.json [hunk 10]
```diff
@@ -1,37 +1,37 @@
...
-  "additional-reading-a-developers-intro": "An introduction to Ethereum for developers",
   "additional-reading-what-is-web3": "What is web3?",
-  "additional-reading-value": "What is value?",
-  "additional-reading-decentralize-everything": "Decentralize everything",
-  "how-do-i-use-ethereum-1": "Using Ethereum is like moving to a new place. Things will seem unfamiliar at first, and everyone moves for their own reasons. Despite this, the first thing you'll need are new keys. The software that creates and holds the keys to your new home is called a \"wallet\".",
+  "additional-reading-ethereum-in-thirty-minutes": "Ethereum in 30 minutes by Vitalik Buterin",
+  "additional-reading-get-eth": "Learn how to get ETH",
+  "how-do-i-use-ethereum-1": "Using Ethereum can mean lots of things to lots of people. Maybe you want to sign in to an app, prove your online identity, participate in a decentralized community, transfer some ETH or swap some tokens. Maybe you want to build something that anyone can use. The first thing you'll need is an account. The easiest way to create and access an account is using software called a wallet.",
```
Comment (jmcook1186) [inline src/intl/en/page-learn.json:29]:
```suggestion
  "how-do-i-use-ethereum-1": "Using Ethereum can mean lots of things to lots of people. Maybe you want to sign in to an app, prove your online identity, or transfer some ETH. The first thing you'll need is an account. The easiest way to create and access an account is using software called a wallet.",
```

**Comment by konopkja** [comment]
@jmcook1186  i noticed currently in preview you changed text to web3 card, but the button still goes to get eth

**Comment by jmcook1186** [comment]
> @jmcook1186 i noticed currently in preview you changed text to web3 card, but the button still goes to get eth

thanks - fixed!

**Review by minimalsm — APPROVED** [review]

**Pull request merged**